xdem/biascorr.py

```diff
@@ -0,0 +1,113 @@
+"""Bias corrections for DEMs"""
+from __future__ import annotations
+
+from typing import Callable, Optional
+
+import numpy as np
+import rasterio as rio
+
+import xdem
+
+
+class DirectionalBias(xdem.coreg.Coreg):
+    """
+    For example for DEM along- or across-track bias correction.
+    """
+
+    def __init__(self, bias_func : Callable = xdem.robust_stats.robust_polynomial_fit):  # pylint: disable=super-init-not-called
+        """
+        Instantiate an directional bias correction object.
+
+        :param bias_func: The function to fit the bias. Default: robust polynomial of degree 1 to 6.
+        """
+        super().__init__(meta={"bias_func": bias_func})
+
+    def _fit_func(self, ref_dem: np.ndarray, tba_dem: np.ndarray, transform: Optional[rio.transform.Affine],
+                  weights: Optional[np.ndarray], angle: Optional[float] = None, verbose: bool = False,**kwargs):
```

Comment:
`**kwargs` does not exist for `Coreg.fit` which needs to be added. Feel free to do so! Additionally, it needs to be supplied to line 507 in main:
https://github.com/GlacioHack/xdem/blob/21e196ab721cfb1868359ee499e2677810a8e112/xdem/coreg.py#L507